src/connection.rs

```diff
@@ -0,0 +1,27 @@
+use crate::save::SaveData;
```

Comment:
Root directory is getting sort of big. Perhaps move this to a subfolder named 'model' since this is a straight up data object. What do you think? Maybe overkill since we might not get many more data objects to handle? I think we'll eventually have to make one for line etc.